scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -414,8 +414,8 @@ <h6>No causative variants</h6>
           <div class="list-group-item list-group-item-action">
             {% if variant._id %}
               {% do already_displayed_variant_ids.append(variant._id) %}
-              <div class="d-flex align-items-center">
```

Comment:
Align buttons for removing causatives and partial causatives:

<img width="490" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/a7296be8-f0ef-4dfb-b308-00e4202525da">